packages/_flutter_web_build_script/pubspec.yaml

```diff
@@ -0,0 +1,97 @@
+name: _flutter_web_build_script
```

Comment:
Consider adding an explanatory README.md for this package that points at any relevant github issues.

Reply:
Done